src/proxy-ui-api/frontend/src/views/InitialConfiguration/InitialConfiguration.vue

```diff
@@ -0,0 +1,231 @@
+            :saveBusy="pinSaveBusy"
+          />
+        </v-stepper-content>
+      </v-stepper-items>
+    </v-stepper>
+
+    <!-- Confirm dialog for warnings when initializing server -->
+    <warningDialog
+      :dialog="confirmInitWarning"
+      :warnings="warningInfo"
+      localizationParent="initialConfiguration.warning"
+      @cancel="confirmInitWarning = false"
+      @accept="acceptInitWarning()"
+    ></warningDialog>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+import SubViewTitle from '@/components/ui/SubViewTitle.vue';
+import TokenPinStep from './TokenPinStep.vue';
+import ConfigurationAnchorStep from './ConfigurationAnchorStep.vue';
+import WarningDialog from './WarningDialog.vue';
+import OwnerMemberStep from './OwnerMemberStep.vue';
+import { Key, Token } from '@/openapi-types';
+import { RouteName, UsageTypes } from '@/global';
+import * as api from '@/util/api';
+
+export default Vue.extend({
+  components: {
+    SubViewTitle,
+    TokenPinStep,
+    ConfigurationAnchorStep,
+    OwnerMemberStep,
+    WarningDialog,
+  },
+  props: {},
+  computed: {
+    ...mapGetters([
+      'isAnchorImported',
+      'isServerOwnerInitialized',
+      'isServerCodeInitialized',
+    ]),
+  },
+  data() {
+    return {
+      currentStep: 1,
+      pinSaveBusy: false,
+      warningInfo: [],
+      confirmInitWarning: false,
+      requestPayload: undefined as any,
+    };
+  },
+  methods: {
+    configAnchorReady(): void {
+      this.currentStep++;
+    },
+
+    ownerMemberReady(): void {
+      this.currentStep++;
+    },
+
+    tokenPinReady(pin: string): void {
+      this.pinSaveBusy = true;
+
+      this.requestPayload = {
+        software_token_pin: pin,
+        ignore_warnings: false,
+      };
+
+      // If owner member is not already set up add it
+      if (!this.isServerOwnerInitialized) {
+        this.requestPayload.owner_member_class = this.$store.getters.initServerMemberClass;
+        this.requestPayload.owner_member_code = this.$store.getters.initServerMemberCode;
+      }
+
+      // Add security code if it's not already set up
+      if (!this.isServerCodeInitialized) {
+        this.requestPayload.security_server_code = this.$store.getters.initServerSSCode;
+      }
+
+      this.initServer(this.requestPayload);
+    },
+
+    acceptInitWarning(): void {
+      this.requestPayload.ignore_warnings = true;
+      this.confirmInitWarning = false;
+      this.initServer(this.requestPayload);
+    },
+
+    initServer(payload: any): void {
+      api
+        .post('/initialization', payload)
+        .then((res) => {
+          this.$store.dispatch('showSuccess', 'initialConfiguration.success');
+          this.pinSaveBusy = false;
+          this.fetchCurrentSecurityServer();
+          this.$router.replace({ name: RouteName.Clients });
+        })
+        .catch((error) => {
+          if (error.response.data.warnings) {
```

Comment:
Perhaps add optional chaining here as it might possible that error response doesn't include 'data'

Reply:
fixed